### PR TITLE
W-15176045 ACM troubleshooting workaround for vulnerability jh

### PR DESCRIPTION
**Pull request by JennyHajee**
# Writer's Quality Checklist

Before merging your PR, did you:

- [ ] Run spell checker
- [ ] Run link checker to check for broken xrefs
- [ ] Check for orphan files
- [ ] Perform a local build and do a final visual check of your content, including checking for:
  - Broken images
  - Dead links
  - Correct rendering of partials if they are used in your content
  - Formatting issues, such as:
    - Misnumbered ordered lists (steps) or incorrectly nested unordered lists
    - Messed up tables
    - Proper indentation
    - Correct header levels
- [ ] Receive final review and signoff from:
  - Technical SME
  - Product Manager
  - Editor or peer reviewer
  - Reporter, if this content is in response to a reported issue (internal or external feedback)
- [ ] If applicable, verify that the software actually got released

### CODE REVIEW

**Review by ferferrarazzo — DISMISSED** [review]

File: modules/ROOT/pages/troubleshooting-avoid-a-potential-security-vulnerability.adoc [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+include::_attributes.adoc[]
+endif::[]
+:imagesdir: ../assets/images/
+
+
+The guest user profile contains the Exchange Viewer permission that gives access to all assets in Exchange with this visibility. An impersonated guest user can potentially exploit this visibility and access assets that are not intended for them. 
+
+To troubleshoot this issue, ensure that anonymous guest users and community users can only access intended and required assets.
+
+. Create a new team in Access Management called Guest Community Users and create a new mapping for External Idp Groups called Community Guest.
```
Comment (Crispy-Salesforce) [inline modules/ROOT/pages/troubleshooting-avoid-a-potential-security-vulnerability.adoc:12]: Is "Idp" spelled like thin on purpose? Or should it be "IDP"?

File: modules/ROOT/pages/troubleshooting-avoid-a-potential-security-vulnerability.adoc [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+. In Salesforce, change the Named Credential authentication information.
+. In Exchange, share the assets with the Guest Community Users team.
+. In Access Management, remove the Exchange Viewer permission and the inherited permissions of the guest user.
+
+To avoid a potential security vulnerability:
+
+. Create a new team in Access Management called Guest Community Users and create a new mapping for External Idp Groups called Community Guest.
```
Comment (Crispy-Salesforce) [inline modules/ROOT/pages/troubleshooting-avoid-a-potential-security-vulnerability.adoc:19]: Is "Idp" spelled like thin on purpose? Or should it be "IDP"?

File: modules/ROOT/pages/troubleshooting-avoid-a-potential-security-vulnerability.adoc [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+.. Go to Access Management.
+.. In the Access Management navigation menu, click *Teams*.
+.. Click *Create Team* and enter `Guest Community Users`.
+.. Select *External Idp Groups* and enter the following fields.
```
Comment (Crispy-Salesforce) [inline modules/ROOT/pages/troubleshooting-avoid-a-potential-security-vulnerability.adoc]:
```suggestion
.. Select *External Idp Groups* and enter the following fields:
```

File: modules/ROOT/pages/troubleshooting-avoid-a-potential-security-vulnerability.adoc [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
++
+* In *Group Name*, enter `Community Guest`.
+* In *Provider Name*, select a provider.
+.. Click *Save Changes*.
+. In Salesforce, change the Named Credential authentication information.
```
Comment (Crispy-Salesforce) [inline modules/ROOT/pages/troubleshooting-avoid-a-potential-security-vulnerability.adoc]:
```suggestion
. In Salesforce, change the Named Credential authentication information:
```

**Review by Crispy-Salesforce — DISMISSED** [review]

**Review by ferferrarazzo — APPROVED** [review]

**Review by Crispy-Salesforce — APPROVED** [review]

**Pull request closed without merging**